server/routes/index.js

```diff
@@ -329,6 +329,30 @@ router.get('/totalCounts', async (req, res, next) => {
   }
 })
 
+router.get('/sampleSignUps', async (req, res, next) => {  
```

Comment:
Can you add a line describing what this route is for?

Reply:
addressed